merlin-framework/src/main/java/gov/nasa/jpl/aerie/merlin/framework/RootModel.java

```diff
@@ -6,7 +6,7 @@
 /**
  * The top-level mission model managed by the Merlin framework.
  */
-public record RootModel<Model>(Model model, ExecutorService executor) implements AutoCloseable {
+public record RootModel<Registry, Model>(Model model, Registry registry, ExecutorService executor) implements AutoCloseable {
   public RootModel {}
```

Comment:
This was here before this PR, but now I'm curious what the purpose of an empty compact constructor is. Should we delete it?

Reply:
🤔 Based on my understanding, we should be able to remove it. I'd rather leave it in for now, simply because RootModel itself might become a generated class as I try to unify the two `Scoped` variables in a mission model. We'll see...